samples/nfc/tag_reader/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _nfc_tag_reader:
+
+NFC: Tag Reader
+###################
+
+The NFC Tag Reader sample demonstrates how to use the :ref:`st25r3911b_nfc_readme` to interract with NFC-A Tag.
+
+Overview
+********
+The samples uses ST25R3911B NFC Reader to read data from the tag in NFC-A technology and shows
+how to use this NFC Reader. The reading procedure is preceded by the Technology Detection which sending of appropriate initial commands (ALL Request, SENS Request)
+and the procedure of Collision Resolution.
+
+Supported tag types:
+
+* NFC Tag 2 Type
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+
+* NFC reader ST25R3911B Nucleo expansion board
+* NFC Type 2 Tag
+
+Building and running
+********************
+
+This sample can be found under :file:`samples/nfc/tag_reader` in the |NCS| folder structure.
+
+See :ref:`gs_programming` for information about how to build and program the application.
+
+Testing
+=======
+After programming the sample to your board, you can test it with NFC-A Type 2 Tag.
+
+1. Connect the board wit :ref:`nfc_tag_reader` application to the terminal (for example, PuTTY).
+   See :ref:`putty` for the required settings.
+#. Touch the ST25R3911B NFC Reader with a Type 2 Tag.
```

Comment:
Do I need to connect the Nucleo board to the DK?

Reply:
Yes, You have to have this board connected to the DK

Reply:
ok, updated